index.js

```diff
@@ -21,12 +22,15 @@ Module._nodeModulePaths = function (from) {
 
 var oldResolveFilename = Module._resolveFilename
 Module._resolveFilename = function (request, parent, isMain) {
-  for (var alias in moduleAliases) {
+  for (var i = moduleAliasNames.length; i-- > 0;) {
```

Comment:
Reverse sorting is faster than forward sorting, which is just fine since it means we don't need to do an `Array.prototype.reverse()` call on line 93.